<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputParameters</class>
 <widget class="QGroupBox" name="InputParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Parameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>dt</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="f_optimize_dt">
       <property name="text">
        <string>Optimise dt</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Current [Î¼A]</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QDoubleSpinBox" name="f_current">
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
       <property name="value">
        <double>0.059000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QDoubleSpinBox" name="f_dt">
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>103</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>12</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="f_error">
       <property name="text">
        <string>---</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Error</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
